### ISSUE TITLE
Add WebC as a template language to core

### ISSUE THREAD

**Issue by darthmall** [11ty/eleventy#2840]
### Describe the solution you'd like

Eleventy supports a number of template languages out of the box. It seems only natural to me that one of the core template languages would be WebC, rather than having it as a plugin.

**Comment by zachleat** [comment]
Yeah, 100%. I’d like to get WebC at 1.0 before this happens. Per #1103 I think Pug is going to move to plugin-land, which will make more than enough room in our node_modules budget.

**Comment by zachleat** [comment]
Punting this one to 4.0

**Comment by briancarbone** [comment]
Does the bundler now included in core conflict with the bundle plugin plugin used in the WebC plugin for Eleventy? I think that uses 2.0.2.

**Comment by vanyauhalin** [comment]
Just 2 cents. Over time, many engines, languages have moved away from core into plugins. And I like the idea of continuing this movement, extracting all of them (see https://github.com/11ty/eleventy/issues/3647). Adding WebC to core looks like a step back.

**Comment by zachleat** [comment]
@vanyauhalin a bit of time and broader thought has me agreeing with this sentiment. I’m going to close this one. Folks can make the case for inclusion if they feel strongly about it but I think keeping core lean and mean (and decoupled) has greater long term benefit
